.gitignore

```diff
@@ -3,5 +3,5 @@ boot/
 lib_managed/
 src_managed/
 project/plugins/project/
-
+.idea/
 .DS_STORE
```

Comment:
Maybe it would be better not to have tool-dependent files in the project gitignore (same for DS_STORE and idea). A user can have a global gitignore in his home directory to handle these files. I do not really mind, but what do you guys prefer?

Reply:
It seems to me like it is easier to get started. It is not critical though

Reply:
*easier to get started* is a good argument, people can still decide to do it otherwise in their own projects.